README.md

```diff
@@ -14,12 +14,29 @@
 
 </p>
 
-NEO: A distributed network for the Smart Economy
+NEO 3.0 (under development): A distributed network for the Smart Economy
 ================
 
 NEO uses digital identity and blockchain technology to digitize assets and leverages smart contracts for autonomously managed digital assets to create a "smart economy" within a decentralized network.
 
-To learn more about NEO, please read the [White Paper](http://docs.neo.org/en-us/index.html)|[白皮书](http://docs.neo.org/zh-cn/index.html).
+To learn more about NEO, please read the [White Paper](https://docs.neo.org/en-us/whitepaper.html)|[白皮书](https://docs.neo.org/zh-cn/whitepaper.html).
```

Comment:
```suggestion
To learn more about NEO, please read the [White Paper](https://docs.neo.org/en-us/whitepaper.html)|[白皮书](https://docs.neo.org/en-us/whitepaper.htmll).
```

Reply:
I think that this was not really a good suggestion...aehuaheuaheauea I will revert it.